homepage/homepage/app/docs/[...slug]/schemas/accounts.mdx

```diff
@@ -0,0 +1,87 @@
+import { CodeGroup } from "@/components/forMdx";
+
+# Accounts & migrations
+
+Accounts are a fundamental building block for user identity and data organization in Jazz applications, providing a structured way to manage both public and private user data while maintaining type-safety and access control.
+
+Account data and metadata are publicly visible to other users.
+
+For private data, we suggest nesting privately owned CoValues within the account root.
+
+## Accounts
+
+To create a custom account type, extend the `Account` class and register it with the Jazz runtime.
+
+<CodeGroup>
+```ts
+import { Account, CoMap } from "jazz-tools"; 
+
+class MyAccountRoot extends CoMap {
+  privateString = co.string; // Readable by the account owner
+}
+
+class MyProfile extends Profile {
+  publicString = co.string; // Readable by everyone
+}
+
+class MyAccount extends Account {
+  root = co.ref(MyAccountRoot);
+  profile = co.ref(MyProfile);
+}
+```
```

Comment:
In this code snippet (without the migration yet), root and profile would both be public right?

Reply:
If you only read this part, I think it gets confusing. I suggest to write only about the Profile. 

In the Migrations section, we can talk about adding root, and how that's where private data should go.